SPECS/rust/rust.spec

```
@@ -73,12 +73,13 @@ mv %{SOURCE7} "$BUILD_CACHE_DIR"
 export CFLAGS="`echo " %{build_cflags} " | sed 's/ -g//'`"
 export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
 
-sh ./configure --prefix=%{_prefix} --enable-extended --tools="cargo"
+sh ./configure --prefix=%{_prefix} --enable-extended --tools="cargo,rustfmt"
 # SUDO_USER=root bypasses a check in the python bootstrap that
 # makes rust refuse to pull sources from the internet
 USER=root SUDO_USER=root %make_build
 
 %check
+ln -s /usr/src/mariner/BUILD/rustc-1.56.1-src/build/x86_64-unknown-linux-gnu/stage2-tools-bin/rustfmt /usr/src/mariner/BUILD/rustc-1.56.1-src/build/x86_64-unknown-linux-gnu/stage0/bin/
```

Ohh wow .. so we were missing `rustfmt` and thus maybe not even going to the test!

How about we drop the `%{_buildroot}/rust...` prefix, so that we survive upgrades:
```suggestion
ln -s ./build/x86_64-unknown-linux-gnu/stage2-tools-bin/rustfmt ./build/x86_64-unknown-linux-gnu/stage0/bin/
```

---

Let me check whether it works before updating. Rust build takes some time.

---

This didn't work as as softlink is created using relative path. Relative path in the build directory and destination directory are not same. IMO using absolute path is better.